ide/vsc/syntax-highligthing/README.md

```diff
@@ -0,0 +1,65 @@
+# ju README
+
+This is the README for your extension "ju". After writing up a brief description, we recommend including the following sections.
+
+## Features
+
+Describe specific features of your extension including screenshots of your extension in action. Image paths are relative to this README file.
```

Comment:
These should be filled out before the release on marketplace I think?

Reply:
We can update that later too.